ocaml/sdk-gen/csharp/templates/XenServer.csproj.mustache

```diff
@@ -1,10 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <Version>@SDK_VERSION@</Version>
     <TargetFramework>netstandard2.0</TargetFramework>
     <OutputType>Library</OutputType>
     <RootNamespace>XenAPI</RootNamespace>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <Title>XenServer.NET</Title>
+    <Authors>Citrix Systems, Inc.</Authors>

```

Comment:
Or even leave them empty if it's possible to set these properties from the msbuild CLI.

Reply:
yes but I left it because we already have the values explicitly specified for the [Java SDK](https://github.com/xapi-project/xen-api/blob/master/ocaml/sdk-gen/java/autogen/xen-api/pom.xml)

Worth changing when we next change these values? as a separate task/PR

Reply:
Ah ok, then leave it for now then and we can revisit both later.